app/views/layouts/_footer.html.erb

```diff
@@ -1,4 +1,3 @@
-
-    <div class="footer">
-      <p>Powered by <a href="http://www.agileventures.org">AgileVentures</a> volunteers &#9734; Open Source software</p>
-    </div>
+<div class="footer">
+  <p>Powered by <a href="http://www.agileventures.org">AgileVentures</a> volunteers &#9734; Open Source software</p>
+</div>
```

Comment:
If the team thinks it's a good idea. I didn't mean for it to be a part of this PR, just wrote down my thoughts - should have made that clear.